### DISCUSSION TITLE
How to use postgresql

### DISCUSSION THREAD

**Discussion by Dostonlv** [prisma/prisma-client-go#991] [Q&A]
I couldn't use postgresql on go-prism. codebase if you drop it or give direction

**Comment by steebchen** [comment]
Please provide some details. You have to use `postgresql` as the provider and provide a postgresql connection string. For example, follow the [quickstart tutorial](https://goprisma.org/docs/getting-started/quickstart), run a postgres instance on your machine, and use the following schema (you might need to adapt username/password/database name):


```prisma
datasource db {
  provider = "postgresql"
  url      = "postgresql://postgres:pw@localhost:5432/db"
}

generator db {
  provider = "go run github.com/steebchen/prisma-client-go"
}

model User {
  id        String   @id @default(uuid())
  name      String?
  email     String
}
```
